### PR TITLE
Create storage for multiple classes

### PR DESCRIPTION
**Pull request by ooimingsheng**
Create ability to store new classes as data

### CODE REVIEW

File: src/test/java/seedu/address/model/DataTest.java [hunk 1]
```diff
@@ -49,52 +49,52 @@ public void resetData_withDuplicatePersons_throwsDuplicatePersonException() {
         Person editedAlice = new PersonBuilder(ALICE).withAddress(VALID_ADDRESS_BOB).withTags(VALID_TAG_HUSBAND)
```
Comment (belbel98) [inline src/test/java/seedu/address/model/DataTest.java]: Does our person class have an **Address**, and we wont be using tags right?

File: src/test/java/seedu/address/testutil/DataBuilder.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
+package seedu.address.testutil;
+
+import seedu.address.model.Data;
+import seedu.address.model.competition.Competition;
+import seedu.address.model.person.Person;
+
+/**
+ * A utility class to help with building Addressbook objects.
```
Comment (belbel98) [inline src/test/java/seedu/address/testutil/DataBuilder.java]: Might need to update comment.

File: src/test/java/seedu/address/testutil/CompetitionBuilder.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
+package seedu.address.testutil;
+
+import java.util.Date;
+
+import seedu.address.model.competition.Competition;
+import seedu.address.model.person.Name;
+
+/**
+ * A utility class to help with building Person objects.
```
Comment (belbel98) [inline src/test/java/seedu/address/testutil/CompetitionBuilder.java]: change `Person` to `Competition` in the comments

File: src/test/java/seedu/address/model/DataTest.java (hunk 1 above)
Comment (ooimingsheng) [reply src/test/java/seedu/address/model/DataTest.java]: this will be changed when @HoWeiChin merges the changes for the person

**Pull request merged**